src/test/java/com/mfarag/todo/TasksTest.java

```diff
@@ -0,0 +1,55 @@
+package com.mfarag.todo;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class TasksTest {
+    private Task TEST_TASK_1 = new Task ("New Task 1") ;
```

Comment:
Just need to break the dependency on the two constants. Use `@Before` setup method and set `TEST_TASK_1` and `TEST_TASK_2` as variables instead of constants and construct them in the setup method.

Ask me if you need more explanation